content/en/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -60,6 +60,12 @@ metadata:
   ...
 ```
 
+{{< note >}}
+Owner references allow resources in the same namespace as the dependents, or
+cluster-scoped resources. However, cross-namespace owner references is
+disallowed by design.
```

Comment:
Is this more clear:

Cluster-scoped owner may have namespaced dependents or cluster-scoped dependents.
Namespaced owner may only have namespaced dependents in the same namespace.

Reply:
👍